src/data/sfsgoals.cfg

```diff
@@ -15,15 +15,45 @@
 # Custom goals can be defined with or without labels.
 #
 # EXAMPLES:
-#
-#  3 3 : _ _ _                                 # one of the default goals (three copies anywhere)
-#  8 not_important_file : _                    # only one copy
+# One of the default goals (three copies anywhere)
+#  3 3 : _ _ _
+
+# Only one copy
+#  8 not_important_file : _
+
+# Two copies
 # 11 important_file : _ _
-# 12 local_copy_on_mars : mars _               # at least one copy in the Martian datacenter
-# 13 cached_on_ssd : ssd _
-# 14 very_important_file : _ _ _ _
-# 15 min_two_locations: _ locationA locationB  # one copy in A, one in B, third anywhere
-# 16 fast_access      : ssd _ _                # one copy on ssd, two additional on any drives
-# 17 two_manufacturers: WD HT                  # one on WD disk, one on HT disk
-# 18 default_ec : $ec(5,2)                     # erasure code with 5 data parts and 2 parity parts
-# 19 wide_ec : $ec(17,9) { ssd ssd }           # erasure code 17+9 with at least two parts on ssd
+
+# Four copies
+# 12 very_important_file : _ _ _ _
+
+# At least one copy in the chunkserver labeled 'mars', and second copy
+# anywhere (including any other chunkserver labelled mars)
+# 12 local_copy_on_mars : mars _
+
+# One copy on the chunkserver labeled 'ssd', and second on 'hdd'
+# 13 cached_on_ssd : ssd hdd
+
+# One copy in A, one in B, third anywhere
+# 15 min_two_locations: _ locationA locationB
+
+# One copy on 'ssd' chunkserver, two on any chunkserver
+# 16 fast_access      : ssd _ _
+
+# One on WD labeled chunkserver, one on HT labeled chunkserver
+# 17 two_manufacturers: WD HT
+
+# EC examples:
+
+# Erasure code with 5 data parts and 2 parity parts on 7 different chunkservers
+# 18 default_ec : $ec(5,2)
+
+# Erasure code with 3 data parts on chunkservers with label 'ssd', and two
```

Comment:
This is not true. The description should be "3 parts in chunkservers labeled 'ssd' and the two other parts in 'hdd'", i.e the order of the labels is not important.